metadata/policy.go

```diff
@@ -94,14 +94,27 @@ func (err *ErrNotEncrypted) Error() string {
 	return fmt.Sprintf("file or directory %q is not encrypted", err.Path)
 }
 
-func policyIoctl(file *os.File, request uintptr, arg unsafe.Pointer) error {
+func getPolicyIoctl(file *os.File, request uintptr, arg unsafe.Pointer) error {
 	_, _, errno := unix.Syscall(unix.SYS_IOCTL, file.Fd(), request, uintptr(arg))
 	if errno == 0 {
 		return nil
 	}
 	return errno
 }
 
+func setPolicyIoctl(file *os.File, request uintptr, arg unsafe.Pointer) error {
```

Comment:
`request` is always `unix.FS_IOC_SET_ENCRYPTION_POLICY` here, so please remove the unneeded parameter and rename this function to `setPolicy()`.

Reply:
done